examples/with-jest/jest.config.ts

```diff
@@ -3,24 +3,7 @@ import type { Config } from 'jest'
 export default {
   rootDir: __dirname,
   setupFilesAfterEnv: ['<rootDir>/jest.setup.ts'],
```

Comment:
This example doesn't really need a setup anymore.

Reply:
It does need it. The setup file sets up MSW globaly, as well as resetHandlers() after each test.

Reply:
You're right!